### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by anchpop**
Thank you to @peterpeterparker for pointing these issues out on the forum

### CODE REVIEW

File: example_sns_init.yaml [hunk 1]
```diff
@@ -83,19 +83,23 @@ description: >
 # cropped to a circle.
 logo: logo.png
 
-# URL to the dapp controlled by the SNS project.
-# Must be a string from 10 to 512 bytes.
-url: https://forum.dfinity.org/thread-where-this-sns-is-discussed
+# URL that represents the SNS, usually a dapp controlled by the SNS project.
+# Must be a string from 10 to 512 bytes, and the domain must be HTTPS.
+# Must be a string of 10 to 512 bytes.
```
Comment (dylancm4) [inline example_sns_init.yaml:88]: Did you mean to repeat this text?

Comment (anchpop) [reply example_sns_init.yaml]: fixed!

File: example_sns_init.yaml [hunk 2]
```diff
@@ -83,19 +83,22 @@ description: >
...
+url: https://mydapp.com
 
 # Metadata for the NNS proposal required to create the SNS. This data will be
 # shown only in the NNS proposal.
 NnsProposal:
     # The title of the NNS proposal. Must be a string of 4 to 256 bytes.
     title: "NNS Proposal to create an SNS named 'Rock Out'"
 
-    # The HTTPS address of additional content required to evaluate the NNS
+    # The HTTPS address of a forum post with additional context required to evaluate the NNS
```
Comment (aterga) [inline example_sns_init.yaml:96]: does not have to be forum post, could be white paper, I think. Cc @LaraAS

**Review by mraszyk — APPROVED** [review]

File: example_sns_init.yaml (hunk 2 above)
Comment (aterga) [reply example_sns_init.yaml:96]: I now see that we have `allowed_domains` set to "forum.dfinity.org". So nevermind.

**Review by aterga — APPROVED** [review]

**Pull request merged**